admin/src/services/api.js

```diff
@@ -3,6 +3,7 @@ import fetchRetry from "fetch-retry";
 import { capture } from "../sentry";
 import { apiURL } from "../config";
 import { createFormDataForFileUpload } from "snu-lib";
+import { ERRORS } from "snu-lib/errors";
```

Comment:
@naustra tu es sûr que l'import fonctionne de cette façon et non :
```suggestion
import { ERRORS } from "snu-lib";
```

Reply:
c'est le back qui kiffe pas a cause du commonJS